scripts/precommit

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-
-npm run lint
```

Comment:
Why not lint and prettify before commiting?

Reply:
added `"precommit": "./scripts/test"` to package.json

Reply:
We decided against `./scripts/test`ing on commit because it discourages frequent commits